ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -2032,6 +2226,12 @@ objects:
             export CLOUDSDK_CONFIG=${HOME}/shared/gcloudconfig
             openshift-install --dir ${HOME}/artifacts/installer destroy cluster
             gcloud deployment-manager deployments delete -q ${CLUSTER_NAME}-{worker,control-plane,bootstrap,security,infra,vpc}
+          elif [[ "${CLUSTER_TYPE}" == "azure4" ]]; then
+            openshift-install --dir=/tmp/artifacts/installer destroy cluster
+            if [[ -f /tmp/artifacts/installer/metadata.json ]]; then
+              export INFRA_ID=`cat /tmp/artifacts/installer/metadata.json | jq -r '.infraID'`
```

Comment:
nit: syntax used elsewhere is `INFRA_ID="$(jq -r .infraID /tmp/artifacts/installer/metadata.json)"`

Reply:
Fixed.